### DISCUSSION TITLE
.

### DISCUSSION THREAD

**Discussion by dandan64** [RayTracing/raytracing.github.io#1666] [Debugging]
the ppm file I get is square compared to the image in the original file which is rectangle 
did someone encounter this problem and has solved it?

**Comment by hollasch** [comment]
Which PPM file are you talking about? If you open up the PPM file in a text editor, you'll see the width and height on the second line. If the image is square, then those values must be equal. I would just go to your code and look at the place where you write those values out, and then figure out from there why the values are not what you expect.
